boards/arm/nrf5340_audio_dk_nrf5340/dts/bindings/nrf5340_audio_dk.yaml

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  This file contains specific pin mappings for the nRF5340 Audio DK.
+
+compatible: "nrf5340_audio_dk"
+
+include: base.yaml
+
+properties:
+    hw-codec-in-gpios:
+      type: phandle-array
+      required: true
+      description: GPIO from HW codec
+
+    hw-codec-irq-in-gpios:
+      type: phandle-array
+      required: true
+      description: Interrupt from HW codec
+
+    hw-codec-reset-out-gpios:
+        type: phandle-array
+        required: true
+        description: Reset HW codec
+
+    hw-codec-sel-out-gpios:
+      type: phandle-array
```

Comment:
Alignment here changes, use 4 space for main indent then 2 additionally for extra indentation